### PR TITLE
VACMS-5775: Update meta titles.

### LINKED ISSUE #5775: Improve metatag for titles and images
## User Story or Problem Statement

<details>
<summary>Screenshot</summary>

metatag_config

</details>

## Acceptance Criteria

- [ ] The default Content title (and any other meta title fields like twitter_cards_title should be `[node:title] |  Veterans Affairs` not `[node:title] | [node:field_office] | Veterans Affairs`. 
  - This is causing a FE Defect: Go to https://www.va.gov/initiatives/recognizing-lgbtq-veterans-during-pride-month/ , https://www.va.gov/resources/can-i-be-buried-in-arlington-national-cemetery/, or any page based on a content type that doesn't have a field_office value (which is required on some content types). See that there is a double pipe in the meta title. 
  - [ ]  Any content type that has a required field_office should be `[node:title] | [node:field_office] | Veterans Affairs`
- [ ]  Every content type that has a "primary" image field should have an og:image specified, using the 2:1 medium image style. This includes:
  - [ ] Event
  - [ ] All VAMC and Vet Center facility types
  - [ ] VAMC system 
  - [ ] Story
  - [ ] Staff profile
  - [ ] Campaign Landing Page **priority**
  - [ ] Benefit Hub Landing Page
  - [ ] Any others with an image field. 

## Implementation steps
- [ ] Update metatag config.

## Design principles

Which [CMS design principle](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/cms/cms-product-design-principles.md) is at play?
- [ ] Purpose-driven: Creates an opportunity to involve the editor community in VA’s mission and content strategy goals.
- [ ] Efficient: Removes distractions and create clear, straightforward paths to get the job done.
- [ ] Approachable: Offers friendly guidance over authoritative instruction.
- [x] Consistent: Reduce user’s mental load by allowing them to fall back on pattern recognition to complete tasks.
- [ ] Empowering: Provide clear information to help editors make decisions about their work.

### CMS Team

Please leave only the team that will do this work selected. If you're not sure, it's fine to leave both selected.

- [ ] `Core Application Team`
- [x] `Product Support Team`

### PR DESCRIPTION
**Pull request by ethanteague**
## Description
Closes #5775

## Testing done
Behat

## QA steps
- Go to `admin/config/search/metatag`
- Visually verify that there are no instances of `og_image: '[node:field_media:entity:image:entity:url]'` remaining
- Visually verify that default content tag for og image is set under `Content` to `og_image: '[node:field_promo:entity:image:entity:3_2_medium_thumbnail:url]'`
- Visually verify all instances of `title: '[node:title] | [node:field_office] | Veterans Affairs'` have been replaced with `title: '[node:title] | Veterans Affairs'`

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [ ] `Core Application Team`
- [x] `Product Support Team`

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change? 
- [ ] Yes, and it's written in issue ____ and queued for publication. 
  - [ ] Merge and ping @ rachel-kauff so she's ready to publish after deployment
- [ ] Yes, but it hasn't yet been written 
  - [ ] Don't merge yet -- ping @ rachel-kauff to prompt her to write and queue content
- [ ] No announcement is needed for this code change. 
  - [ ] Merge & carry on unburdened by announcements

### CODE REVIEW

File: config/sync/metatag.metatag_defaults.node.yml [hunk 1]
```diff
@@ -7,15 +7,15 @@ _core:
...
-  title: '[node:title] | [node:field_office] | Veterans Affairs'
+  title: '[node:title] | Veterans Affairs'
   description: '[node:field_description:value]'
   image_src: '[node:field_media:entity:image:entity:url]'
   og_image_alt: '[node:field_media:entity:image:alt]'
-  og_image: '[node:field_media:entity:image:entity:url]'
+  og_image: '[node:field_media:entity:image:entity:3_2_medium_thumbnail:url]'
   og_description: '[node:field_description:value]'
   og_site_name: 'Veterans Affairs'
-  og_title: '[node:title] | [node:field_office] | Veterans Affairs'
```
Comment (kevwalsh) [inline config/sync/metatag.metatag_defaults.node.yml:17]: We may need to remove the og_image spec altogether, pending convo with PW. Ideally they would check for a value before printing it.

File: config/sync/metatag.metatag_defaults.node__event.yml [hunk 2]
```diff
@@ -6,8 +6,7 @@ id: node__event
 label: 'Content: Event'
 tags:
   title: '[node:field_office] | [node:title] | Veterans Affairs'
-  og_image: '[node:field_media:entity:image:entity:url]'
   og_image_height: '314'
-  og_title: '[node:title] | [node:field_office] | Veterans Affairs'
+  og_title: '[node:title] | Veterans Affairs'
   twitter_cards_image: '[node:field_media:entity:image:entity:url]'
-  twitter_cards_title: '[node:title] | [node:field_office] | Veterans Affairs'
```
Comment (kevwalsh) [inline config/sync/metatag.metatag_defaults.node__event.yml:13]: We need to keep `[node:title] | [node:field_office] | Veterans Affairs` wherever field_office is required.

File: config/sync/metatag.metatag_defaults.node__event_listing.yml [hunk 3]
```diff
@@ -5,5 +5,5 @@ dependencies: {  }
 id: node__event_listing
 label: 'Content: Event listing'
 tags:
-  title: '[node:title] | [node:field_office] | Veterans Affairs'
-  twitter_cards_title: '[node:title] | [node:field_office] | Veterans Affairs'
```
Comment (kevwalsh) [inline config/sync/metatag.metatag_defaults.node__event_listing.yml:9]: We need to keep `[node:title] | [node:field_office] | Veterans Affairs` wherever field_office is required.

File: config/sync/metatag.metatag_defaults.node__health_care_local_facility.yml [hunk 4]
```diff
@@ -5,5 +5,5 @@ dependencies: {  }
 id: node__health_care_local_facility
 label: 'Content: Health Care Local Facility'
 tags:
-  title: '[node:title] | [node:field_office] | Veterans Affairs'
-  twitter_cards_title: '[node:title] | [node:field_office] | Veterans Affairs'
```
Comment (kevwalsh) [inline config/sync/metatag.metatag_defaults.node__health_care_local_facility.yml:9]: We need to keep `[node:title] | [node:field_office] | Veterans Affairs` wherever field_office is required.

File: config/sync/metatag.metatag_defaults.node__vamc_operating_status_and_alerts.yml [hunk 5]
```diff
@@ -5,6 +5,6 @@ dependencies: {  }
 id: node__vamc_operating_status_and_alerts
 label: 'Content: VAMC operating status'
 tags:
-  title: '[node:title] | [node:field_office] | Veterans Affairs'
+  title: '[node:title] | Veterans Affairs'
   description: '[node:field_office] facility operating statuses and emergency information.'
-  twitter_cards_title: '[node:title] | [node:field_office] | Veterans Affairs'
```
Comment (kevwalsh) [inline config/sync/metatag.metatag_defaults.node__vamc_operating_status_and_alerts.yml:10]: We need to keep `[node:title] | [node:field_office] | Veterans Affairs` wherever field_office is required.

File: config/sync/metatag.metatag_defaults.node__press_release.yml [hunk 6]
```diff
@@ -5,9 +5,9 @@ dependencies: {  }
...
 label: 'Content: Press release'
 tags:
-  title: '[node:title] | [node:field_office] | Veterans Affairs'
+  title: '[node:title] | Veterans Affairs'
   description: '[node:field_intro_text]'
   og_description: '[node:field_intro_text]'
-  og_title: '[node:title] | [node:field_office] | Veterans Affairs'
+  og_title: '[node:title] | Veterans Affairs'
   twitter_cards_description: '[node:field_intro_text]'
-  twitter_cards_title: '[node:title] | [node:field_office] | Veterans Affairs'
```
Comment (kevwalsh) [inline config/sync/metatag.metatag_defaults.node__press_release.yml:13]: We need to keep `[node:title] | [node:field_office] | Veterans Affairs` wherever field_office is required.

File: config/sync/metatag.metatag_defaults.node__person_profile.yml [hunk 7]
```diff
@@ -5,9 +5,9 @@ dependencies: {  }
...
 label: 'Content: Staff profile'
 tags:
-  title: '[node:title] | [node:field_office] | Veterans Affairs'
+  title: '[node:title] | Veterans Affairs'
   description: '[node:field_intro_text]'
   og_description: '[node:field_intro_text]'
-  og_title: '[node:title] | [node:field_office] | Veterans Affairs'
+  og_title: '[node:title] | Veterans Affairs'
   twitter_cards_description: '[node:field_intro_text]'
-  twitter_cards_title: '[node:title] | [node:field_office] | Veterans Affairs'
```
Comment (kevwalsh) [inline config/sync/metatag.metatag_defaults.node__person_profile.yml:13]: We need to keep `[node:title] | [node:field_office] | Veterans Affairs` wherever field_office is required.

File: config/sync/metatag.metatag_defaults.node__news_story.yml [hunk 8]
```diff
@@ -5,9 +5,9 @@ dependencies: {  }
 id: node__news_story
 label: 'Content: News story'
 tags:
-  title: '[node:title] | [node:field_office] | Veterans Affairs'
+  title: '[node:title] | Veterans Affairs'
   description: '[node:field_intro_text]'
   og_description: '[node:field_intro_text]'
-  og_title: '[node:title] | [node:field_office] | Veterans Affairs'
```
Comment (kevwalsh) [inline config/sync/metatag.metatag_defaults.node__news_story.yml:11]: We need to keep `[node:title] | [node:field_office] | Veterans Affairs` wherever field_office is required.

File: config/sync/metatag.metatag_defaults.node__health_care_region_page.yml [hunk 9]
```diff
@@ -5,5 +5,5 @@ dependencies: {  }
 id: node__health_care_region_page
 label: 'Content: Health Care System'
 tags:
-  title: '[node:title] | [node:field_office] | Veterans Affairs'
```
Comment (kevwalsh) [inline config/sync/metatag.metatag_defaults.node__health_care_region_page.yml:8]: We need to keep `[node:title] | [node:field_office] | Veterans Affairs` wherever field_office is required.

File: config/sync/metatag.metatag_defaults.node__health_care_region_page.yml [hunk 10]
```diff
@@ -5,5 +5,5 @@ dependencies: {  }
 id: node__health_care_region_page
 label: 'Content: Health Care System'
```
Comment (kevwalsh) [inline config/sync/metatag.metatag_defaults.node__health_care_region_page.yml:6]: This one should have og:image specified.

File: config/sync/metatag.metatag_defaults.node__health_care_region_detail_page.yml [hunk 11]
```diff
@@ -5,5 +5,5 @@ dependencies: {  }
 id: node__health_care_region_detail_page
 label: 'Content: Health Care Region Detail Page'
 tags:
-  title: '[node:title] | [node:field_office] | Veterans Affairs'
-  twitter_cards_title: '[node:title] | [node:field_office] | Veterans Affairs'
```
Comment (kevwalsh) [inline config/sync/metatag.metatag_defaults.node__health_care_region_detail_page.yml:9]: We need to keep `[node:title] | [node:field_office] | Veterans Affairs` wherever field_office is required.

**Review by kevwalsh — CHANGES_REQUESTED** [review]
it's good that our default will no longer include `node:field_office` but we need to keep `[node:title] | [node:field_office] | Veterans Affairs` wherever field_office is required. Sorry if ACs were unclear in #5775, i just updated them a bit.

**Comment by ethanteague** [comment]
Thanks for taking the time to go through all those files, @kevwalsh :heart: - back to you.

File: config/sync/metatag.metatag_defaults.node__campaign_landing_page.yml [hunk 12]
```diff
@@ -0,0 +1,8 @@
+uuid: f15d1d72-95e4-4ada-b295-c2183f604b50
+langcode: en
+status: true
+dependencies: {  }
+id: node__campaign_landing_page
+label: 'Content: Campaign Landing Page'
+tags:
+  og_image: '[node:field_hero_image:entity:image:entity:3_2_medium_thumbnail:url]'
```
Comment (kevwalsh) [inline config/sync/metatag.metatag_defaults.node__campaign_landing_page.yml:8]: NOICE

**Review by kevwalsh — CHANGES_REQUESTED** [review]
I'm really sorry, i feel like i'm fumbling through the specs on this. we probably needed a spreadsheet approach to this one. 

Stories, Events, News releases do not have a field_office, so they'll need to pick that up by going through their respective List pages. See for example https://pr5828-wrb1q983pfwgqw6ixqwofsjszpwdosrp.ci.cms.va.gov/wilmington-health-care/stories/veteran-suicide-prevention-in-new-jersey

Leadership pages have a field_office that will; need to be inserted, see for example 
https://pr5828-wrb1q983pfwgqw6ixqwofsjszpwdosrp.ci.cms.va.gov/butler-health-care/about-us/leadership

Finally, Operating Status we should probalby treat specially, as follows: 
`Operating Status | [node:field_office] | Veterans Affairs` since their [node:title] is weird.

**Comment by ethanteague** [comment]
> I'm really sorry, i feel like i'm fumbling through the specs on this. we probably needed a spreadsheet approach to this one.
> 
> Stories, Events, News releases do not have a field_office, so they'll need to pick that up by going through their respective List pages. See for example https://pr5828-wrb1q983pfwgqw6ixqwofsjszpwdosrp.ci.cms.va.gov/wilmington-health-care/stories/veteran-suicide-prevention-in-new-jersey
> 
> Leadership pages have a field_office that will; need to be inserted, see for example
> https://pr5828-wrb1q983pfwgqw6ixqwofsjszpwdosrp.ci.cms.va.gov/butler-health-care/about-us/leadership
> 
> Finally, Operating Status we should probalby treat specially, as follows:
> `Operating Status | [node:field_office] | Veterans Affairs` since their [node:title] is weird.

Np a'tall, @kevwalsh - I should have thought of that.

**Review by swirtSJW — APPROVED** [review]
Approved by me.  Ready for merge if Kev approves.

**Comment by swirtSJW** [comment]
image

**Pull request merged**